php_apcu/README.md

```diff
@@ -0,0 +1,81 @@
+# Agent Check: php_apcu
+
+## Overview
+
+This check monitors [PHP APCu][1] through the Datadog Agent.
+
+## Setup
+
+Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][2] for guidance on applying these instructions.
+
+### Installation
+
+To install the `php_apcu` check on your host:
+
+
+1. Install the [developer toolkit][3].
+ on any machine.
+
+2. Run `ddev release build php_apcu` to build the package.
+
+3. [Download the Datadog Agent][4].
+
+4. Upload the build artifact to any host with an Agent and
+ run `datadog-agent integration install -w
+ path/to/php_apcu/dist/<ARTIFACT_NAME>.whl`.
+
+#### APCu Setup
+
+Provide a URL to the collection program.
+
+APCu does not have a metric output by itself.
+
+* /opt/datadog-agent/embedded/lib/python3.8/site-packages/datadog_checks/php_apcu/assets/exporter/apcu-dd-handler.php
+
+It will work if you refer to it as a php file, but you may want to configure it as follows
```

Comment:
```suggestion

APCu does not expose metrics by default. This integration includes a metric exporter, located here:

    ```
    /opt/datadog-agent/embedded/lib/python3.8/site-packages/datadog_checks/php_apcu/assets/exporter/apcu-dd-handler.php
    ```

It will work if you refer to it as a php file, but you may want to configure it as follows
```

Let's reword this for clarity

Reply:
thanks 😄